b2share/config.py

```diff
@@ -47,6 +41,180 @@
+
+# I18N
+# ====
+#: Default language
+BABEL_DEFAULT_LANGUAGE = 'en'
+#: Default time zone
+BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
+#: Other supported languages (do not include the default language in list).
+I18N_LANGUAGES = [
+    # ('fr', _('French'))
+]
+
+# Base templates
+# ==============
+#: Global base template.
+BASE_TEMPLATE = 'b2share_main/base.html'
+#: Cover page base template (used for e.g. login/sign-up).
+COVER_TEMPLATE = 'invenio_theme/page_cover.html'
+#: Footer base template.
+MAIN_TEMPLATE = 'invenio_theme/footer.html'
+#: Header base template.
+HEADER_TEMPLATE = 'invenio_theme/header.html'
+#: Settings base template.
+SETTINGS_TEMPLATE = 'invenio_theme/page_settings.html'
+
+# Theme configuration
+# ===================
+#: Site name
+THEME_SITENAME = _('B2SHARE')
+#: Use default frontpage.
+THEME_FRONTPAGE = True
+#: Frontpage title.
+THEME_FRONTPAGE_TITLE = _('B2SHARE')
+#: Frontpage template.
+THEME_FRONTPAGE_TEMPLATE = 'b2share_main/page.html'
+
+# Email configuration
+# ===================
+#: Email address for support.
+SUPPORT_EMAIL = "info@eudat.eu"
+#: Disable email sending by default.
+MAIL_SUPPRESS_SEND = True
+
+# Assets
+# ======
+#: Static files collection method (defaults to copying files).
+COLLECT_STORAGE = 'flask_collect.storage.file'
+
+# Accounts
+# ========
+#: Email address used as sender of account registration emails.
+SECURITY_EMAIL_SENDER = SUPPORT_EMAIL
+#: Email subject for account registration emails.
+SECURITY_EMAIL_SUBJECT_REGISTER = _(
+    "Welcome to B2SHARE!")
+#: Redis session storage URL.
+CACHE_REDIS_URL=os.environ.get("INVENIO_CACHE_REDIS_URL", 'redis://localhost:6379/0')
+ACCOUNTS_SESSION_REDIS_URL = os.environ.get("INVENIO_ACCOUNTS_SESSION_REDIS_URL", 'redis://localhost:6379/1')
+#: Enable session/user id request tracing. This feature will add X-Session-ID
+#: and X-User-ID headers to HTTP response. You MUST ensure that NGINX (or other
+#: proxies) removes these headers again before sending the response to the
+#: client. Set to False, in case of doubt.
+ACCOUNTS_USERINFO_HEADERS = True
+
+# Database
+# ========
+#: Database URI including user and password
+SQLALCHEMY_DATABASE_URI = os.environ.get("INVENIO_SQLALCHEMY_DATABASE_URI", \
+    'postgresql+psycopg2://b2share:b2share@localhost/b2share')
+
+# JSONSchemas
+# ===========
+#: Hostname used in URLs for local JSONSchemas.
+JSONSCHEMAS_HOST = os.environ.get("JSONSCHEMAS_HOST", 'b2share.eudat.eu')
+
+# APPLICATION ROOT
+APPLICATION_ROOT = os.environ.get("APPLICATION_ROOT", '')
+
+# Flask configuration
+# ===================
+# See details on
+# http://flask.pocoo.org/docs/0.12/config/#builtin-configuration-values
+
+#: Secret key - each installation (dev, production, ...) needs a separate key.
+#: It should be changed before deploying.
+SECRET_KEY = 'CHANGE_ME'
+#: Max upload size for form data via application/mulitpart-formdata.
+MAX_CONTENT_LENGTH = 100 * 1024 * 1024  # 100 MiB
```

Comment:
It might be that Invenio has changed the logic behind invenio-files-rest, since this `MAX_CONTENT_LENGTH` has been defined here.
-> **Is `FILES_REST_DEFAULT_MAX_FILE_SIZE` is not used any more?** (https://github.com/EUDAT-B2SHARE/b2share/blob/d846068c71853221ac53239c9efc56e3e795e6c3/b2share/config.py#L406-L409)

HK: Unit tests might benefit from a test that uploads a large test file, maybe generated on-the-fly during tests.